crates/typst/src/foundations/ty.rs

```diff
@@ -148,7 +148,7 @@ impl Debug for Type {
 
 impl Repr for Type {
     fn repr(&self) -> EcoString {
-        self.long_name().into()
+        self.short_name().into()
```

Comment:
short and long names are used in a bunch of different places. it is unfortunate that both need to exist, but ultimately, we want longer, more descriptive names for error message.

it might make sense to change it here, but before doing that I would like to have an overview over all places where the both variants are used to see whether everything ends up in a consistent state.